### PR TITLE
append emma grasmeder

### PR DESCRIPTION
**Pull request by emgrasmeder**
<3

### CODE REVIEW

File: websites.json [hunk 1]
```diff
@@ -29,6 +29,8 @@
                 ["Zuri Hunter","http://zurihunter.com/"],
                 ["Kaitlynn Hendricks","http://philoticconnection.com"],
                 ["Julia Allyce","http://www.juliaallyce.com/"],
-                ["Jessica Bell","http://sirjessthebrave.github.io/"]
+                ["Jessica Bell","http://sirjessthebrave.github.io/"],
+                ["Emma Grasmeder","http://www.github.com/emmagras/about"]
+
```
Comment (alulsh) [inline websites.json]: Code style - remove extra line of whitespace, please? 😅

**Comment by alulsh** [comment]
Also, just merged https://github.com/alulsh/DC-Tech-Women-And-NB-Websites/pull/4 which caused a merge conflict `¯\_(ツ)_/¯`

**Comment by emgrasmeder** [comment]
Wow, I have no idea how to rebase this.

**Comment by alulsh** [comment]
Me neither 😅  merged https://github.com/alulsh/DC-Tech-Women-And-NB-Websites/pull/6 though!

**Pull request closed without merging**